app/services/forest_liana/value_stat_getter.rb

```diff
@@ -4,49 +4,21 @@ class ValueStatGetter < StatGetter
 
     def perform
       return if @params[:aggregate].blank?
-      valueCurrent = get_resource().eager_load(@includes)
-      valuePrevious = get_resource().eager_load(@includes)
-      filter_date_interval = false
+      resource = current_value = get_resource().eager_load(@includes)
```

Comment:
Why keeping two variables

Reply:
if no filters I need current_value to be set